src/main/java/de/adesso/anki/battle/GameEngine.java

```diff
@@ -110,23 +141,31 @@ public void gameLoop() {
             }
 
             
-
+            // Remove while iterating, leads to exception 
+            //java.util.ConcurrentModificationException: in renderer?
+            //collisionHandling();

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)